qa-tests-backend/src/main/groovy/util/SplunkUtil.groovy

```diff
@@ -141,6 +133,26 @@ class SplunkUtil {
         }
     }
 
+    static String createHECToken(int port, String tokenName = "stackrox") {
+        Response response = null
+        withRetry(20, 15) {
+            response =  given().auth()
+                .basic("admin", SPLUNK_ADMIN_PASSWORD)
+                .formParam("name", tokenName)
+                .formParam("output_mode", "json")
+                .post("https://127.0.0.1:${port}/servicesNS/nobody/search/data/inputs/http")
```

Comment:
I think we should assert that the response is valid here.

Reply:
Good idea. Added a check for token length